Cython/Compiler/Code.py

```diff
@@ -2109,6 +2107,17 @@ def put_var_declaration(self, entry, storage_class="",
         self.putln(";")
         self.funcstate.scope.use_entry_utility_code(entry)
 
+    def put_var_declaration_unused_if_needed(self, entry, definition=True):
+        #print "Code.put_var_declaration:", entry.name, "definition =", definition ###
+        if entry.visibility == 'private' and not (definition or entry.defined_in_pxd):
+            #print "...private and not definition, skipping", entry.cname ###
+            return
+        if entry.visibility == "private" and not entry.used:
+            #print "...private and not used, skipping", entry.cname ###
+            return
```

Comment:
```suggestion
        if entry.visibility == 'private' and not (definition or entry.defined_in_pxd):
            return
        if entry.visibility == "private" and not entry.used:
            return
```

Just cleaning up unused `print` statements